ai-context/policy-reference.md

```
@@ -928,30 +928,93 @@ Each policy in the `policies:` array should have a unique `name` and point to it
+    description: Minimum threshold percentage
     default: "80"
-```
+  allowed_values:
+    description: Comma-separated list of allowed values
+    default: "value1,value2"
+```
+
+### Landing Page Field Reference
+
+| Field | Required | Description |
+|-------|----------|-------------|
```

markdownlint-cli2 (0.20.0)

Table column style
Table pipe is missing space to the right for style "compact"

(MD060, table-column-style)